examples/index.json

```diff
@@ -832,6 +832,10 @@
                 "name": "Client Data Binding",
                 "url": "http-client-data-binding"
             },
+            {
```

Comment:
Shouldn't we remove the older bbe from index.json?

Reply:
It's already been removed.

Reply:
Ack